src/Management/test/Endpoint.Test/Actuators/Hypermedia/HypermediaEndpointTest.cs

```diff
@@ -24,6 +24,14 @@ public async Task Invoke_ReturnsExpectedLinks()
             services.AddInfoActuator();
         };
 
+        testContext.AdditionalConfiguration = configuration =>
```

Comment:
How was this test passing before without changing the management path?

Reply:
The original code used the incoming HTTP request URL as input, ignoring what was configured.

Reply:
Sorry, I don't see any related changes in this PR in BaseTest or TestContext that help me understand how it worked before.. where/when did that change?

Reply:
The code in `HypermediaService` changed to read the base path from options instead of appending to the request URL. Formerly, it assumed that a request to `/foobar` meant the base path is `/foobar`. Therefore the unit test needs to provide compatible options now.

Reply:
Oh _wow_, yeah... Good catch there, the old behavior is silly